crates/mod-host-assets/include/dl_allocator.hpp

```diff
@@ -0,0 +1,60 @@
+#ifndef DANTELION_DL_ALLOCATOR_H
```

Comment:
`#pragma once` for include guards, we don't care about standard C++ here.

Reply:
https://github.com/garyttierney/me3/pull/22/commits/3724dbf6abbd846c3750f251d1be7257556e15ab